### PR TITLE
Tackle-804 Subversion configuration for MTA 6.0

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTA 6.0

Resolves: https://issues.redhat.com/browse/TACKLE-804

Preview: https://deploy-preview-564--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-web-subv-rep_ocp-42

### CODE REVIEW

File: docs/web-console-guide/master.adoc [hunk 1]
```diff
@@ -112,6 +112,9 @@ include::topics/web-add-user.adoc[leveloffset=+2]
 
 // TODO: eventually add steps for reverting back to no auth required
```
Comment (istein1) [inline docs/web-console-guide/master.adoc:113]: Is this TODO still relevant or needs to be wiped off?

File: docs/topics/mta-web-subv-rep.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+:_content-type: PROCEDURE
+[id="mta-web-subv-rep_{context}"]
+= Configuring Subversion repositories
+
+
+.Procedure
+
+. In the left menu of *Administrator* view, click *Repositories* and then *Subversion*.
+. Click, *Consume insecure Git repositories* toggle switch to enable.
```
Comment (istein1) [inline docs/topics/mta-web-subv-rep.adoc:13]: I think same content is mentioned here:
https://github.com/windup/windup-documentation/pull/572/files#diff-4afe5b22018a43d3e8bf8012ac5186e81dd02b21b6b1ab9fb953eb67555438fdR15
Can you please check if there is no duplication for repositories configuration altogether?

File: docs/web-console-guide/master.adoc (hunk 1 above)
Comment (CarmiWisemon) [reply docs/web-console-guide/master.adoc]: It seems to already be deleted.
@RichardHoch  is that correct?

File: docs/topics/mta-web-subv-rep.adoc (hunk 2 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-subv-rep.adoc:13]: There are 2 separate sub-categories that look similar, but are different. Richard is working on Configuring credentials and I am working on Configuring repositories. The headings look similar but they are different topics.

File: docs/topics/mta-web-subv-rep.adoc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-subv-rep_{context}"]
+= Configuring Subversion repositories
+
```
Comment (RichardHoch) [inline docs/topics/mta-web-subv-rep.adoc:8]: You need an introductory sentence, even if it's "You can do x in the y view"

File: docs/topics/mta-web-subv-rep.adoc [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+.Procedure
+
+. In the *Administrator* view, click *Repositories* and then *Subversion*.
+. Toggle *Consume insecure Git repositories* to enable.
```
Comment (RichardHoch) [inline docs/topics/mta-web-subv-rep.adoc:13]: To enable what? "Enable" is a transitive verb, so it requires a direct object.

**Comment by RichardHoch** [comment]
1. Remember to click *Ready for review* when you ask for your first SME review. 
2. I made a couple small suggestions; otherwise LGTM.

File: docs/topics/mta-web-subv-rep.adoc (hunk 3 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-subv-rep.adoc:8]: You can configure Subversion repositories with the following steps.

File: docs/topics/mta-web-subv-rep.adoc (hunk 4 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-subv-rep.adoc]: Toggle Consume insecure Git repositories to enable the configuration.

**Pull request merged**